elastic/eui
# [Visual Refresh] Provide legacy JSON tokens via EUI

## Summary

>[!IMPORTANT]
This update is temporary as it's only a workaround while we're working on finding a proper solution to loading the legacy files in Kibana from the new theme package.

This PR adds the legacy JSON files to the eui package.

## QA

- verify files are available:
  - checkout this PR: `gh pr checkout 8125`
  - build the `eui` package locally: `yarn build-pack` (in `/packages/eui`)
  - [ ] check the built `.tgz` package content and verify the added JSON files are available in the `dist` folder
